zoom:
  ros__parameters:
    always_rectify: False
    always_publish: False
    cam_info_period: 5
    flip: False
    fourcc: [M, J, P, G] # 1196444237.0
    frame_id: zoom
    height: 360
    intrinsic: True
    name: zoom
    publish_rate: 10.0
    read_rate: 30.0
    rectify: False
    roi_exposure: False                 # [bool](1):Enable/(0):Disable Using our custom made exposure based on histogram over center ROI
    type: USB
    video_path: ''
    video_stream_recovery_time: 3       # [int][sec] time to try to reconnect camera
    video_stream_recovery_tries: 10     # [int] number of tries to reconnect a camera
    width: 640

left:
  ros__parameters:
    always_rectify: False
    always_publish: False
    cam_info_period: 5
    flip: False
    fourcc: [M, J, P, G] # 1196444237.0
    frame_id: left
    height: 360
    intrinsic: True
    name: left
    publish_rate: 10.0
    read_rate: 30.0
    rectify: True
    double_rectify: True
    roi_exposure: False               # [bool](1):Enable/(0):Disable Using our custom made exposure based on histogram over center ROI
    type: USB
    video_path: ''
    video_stream_recovery_time: 3     # [int][sec] time to try to reconnect camera
    video_stream_recovery_tries: 10   # [int] number of tries to reconnect a camera
    width: 640

right:
  ros__parameters:
    always_rectify: False
    always_publish: False
    cam_info_period: 5
    flip: False
    fourcc: [M, J, P, G] # 1196444237.0
    frame_id: right
    height: 360
    intrinsic: True
    name: right
    publish_rate: 10.0
    read_rate: 30.0
    rectify: True
    double_rectify: True
    roi_exposure: False               # [bool](1):Enable/(0):Disable Using our custom made exposure based on histogram over center ROI
    type: USB
    video_path: ''
    video_stream_recovery_time: 3     # [int][sec] time to try to reconnect camera
    video_stream_recovery_tries: 10   # [int] number of tries to reconnect a camera
    width: 640

rear:
  ros__parameters:
    always_rectify: False
    always_publish: False
    cam_info_period: 5
    flip: False
    fourcc: [M, J, P, G] # 1196444237.0
    frame_id: rear
    height: 360
    intrinsic: True
    name: rear
    publish_rate: 10.0
    read_rate: 30.0
    rectify: False
    roi_exposure: False               # [bool](1):Enable/(0):Disable Using our custom made exposure based on histogram over center ROI
    type: USB
    video_path: ''
    video_stream_recovery_time: 3     # [int][sec] time to try to reconnect camera
    video_stream_recovery_tries: 10   # [int] number of tries to reconnect a camera
    width: 640

inner:
  ros__parameters:
    always_rectify: False
    always_publish: False
    cam_info_period: 5
    flip: False
    fourcc: [M, J, P, G] # 1196444237.0
    frame_id: inner
    height: 240
    intrinsic: False
    name: inner
    publish_rate: 2.0
    read_rate: 20.0
    rectify: False
    roi_exposure: False               # [bool](1):Enable/(0):Disable Using our custom made exposure based on histogram over center ROI
    type: USB
    video_path: ''
    video_stream_recovery_time: 3     # [int][sec] time to try to reconnect camera
    video_stream_recovery_tries: 10   # [int] number of tries to reconnect a camera
    width: 320                        # [float] the inner camera uses different resolution as it is a HD camera and we dont need that resolution
